src/twilio/twilio_constants.bal

```diff
@@ -65,4 +55,4 @@ final string AUTH_TOKEN = "AUTH_TOKEN";
 final string AUTHY_API_KEY = "AUTHY_API_KEY";
 
 //Error codes
-final string TWILIO_ERROR_CODE = "(ballerinax/twilio)TwilioError";
+public const TWILIO_ERROR = "(ballerinax/twilio)TwilioError";
```

Comment:
Do we need this error code anymore?

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerinax-twilio/pull/64/commits/c894bd88aad5f91ac76beb5d3b8a29dac5c67007